server/routerlicious/packages/services-utils/src/redisUtils.ts

```diff
@@ -119,3 +119,7 @@ export const executeRedisMultiWithHmsetExpireAndLpush = async (
 				reject(error);
 			});
 	});
+
+export const getRedisClusterRetryStrategy =
+	(options: { delayPerAttemptMs: number; maxDelayMs: number }) => (attempts: number) =>
+		Math.min(attempts * options.delayPerAttemptMs, options.maxDelayMs);
```

Comment:
Will this retry forever?

Reply:
Nope, it should only retry "maxRetriesPerRequest" number of times, which is currently set at 20